config/testgrids/gcp-guest/gcp-guest.yaml

```diff
@@ -17,11 +17,11 @@ test_groups:
 dashboards:
 - name: google-gcp-guest
   dashboard_tab:
-  - name: OSConfig presubmits - gobuild
+  - name: OSConfig presubmits-gobuild
```

Comment:
Why?

Reply:
there was a unit test faliure error that was happening because of dublicate tab names ... I did this to check if there was an error interpreting these strings... revert these?